File: Beecrowd/3303 Palavrão/main.c
#include <stdio.h>

int main(){
    int i = 0;
    char palavrao[20];

    scanf("%s", palavrao);

    for(i = 0; palavrao[i] != '\0'; i++);

    if(i >= 10)
        printf("palavrao\n");
    else
        printf("palavrinha\n");

    return 0;
}

File: Beecrowd/3303 Palavrão/Main.java
import java.io.IOException;
import java.util.Scanner;

public class Main {

    public static void main(String[] args) throws IOException {
        Scanner sc = new Scanner(System.in);

        String palav;

        palav = sc.next();

        if (palav.length() >= 10)
            System.out.println("palavrao");
        else
            System.out.println("palavrinha");

        sc.close();
    }

}